packages/yoastseo/src/languageProcessing/researches/getWordForms.js

```diff
@@ -176,20 +177,32 @@ function getWordForms( keyphrase, synonyms, allWordsFromPaper, functionWords, st
 /**
  * Gets all matching word forms for the keyphrase and synonyms.
  *
- * @param {Paper}       paper       The paper.
- * @param {Researcher}  researcher  The researcher.
+ * @param {Paper}       paper       							The paper.
+ * @param {Researcher}  researcher  							The researcher.
+ * @param {boolean}  	hyphensAreAlwaysWordBoundaries  		Whether hyphens should be treated as word boundaries for
+ * 																all languages, regardless of the researcher's config.
  *
  * @returns {Object} Object with an array of keyphrase forms and an array of arrays of synonyms forms, based on the forms
  * found in the text or created forms.
  */
-export default function( paper, researcher ) {
+export default function( paper, researcher, hyphensAreAlwaysWordBoundaries = false ) {
```

Comment:
Because for keyphrase in slug assessment in Indonesian we want to ignore the information from the researcher 😅 so hyphens should be word boundaries in that case

Reply:
Thanks, that's such a smart solution that I didn't think of!